lib/Doctrine/ODM/PHPCR/Query/Builder/ConverterPhpcr.php

```diff
@@ -290,7 +290,7 @@ protected function walkOperandStaticLiteral(OperandStaticLiteral $node): Literal
     {
         $value = $node->getValue();
 
-        if ($field = $node->getParent()?->getChildOfType(AbstractNode::NT_OPERAND_DYNAMIC)) {
+        if ($field = $node->getParent()?->getChildOfType(QBConstants::NT_OPERAND_DYNAMIC)) {
```

Comment:
`AbstractNode` seems to be in the same namespace. Neither use is needed nor the alias.